codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/middleware/MiddlewareBuilder.java

```diff
@@ -161,7 +161,9 @@ public void addDefaultMiddleware(GenerationContext context) {
                         transport.getTransportClient().render(context))
                 .operationParams((ctx, operation) -> {
                     Map<String, String> params = new HashMap<>();
+                    Shape outputShape = ctx.getModel().expectShape(operation.getOutputShape());
                     params.put("stub_class", "Stubs::" + symbolProvider.toSymbol(operation).getName());
+                    params.put("stub_params_class", "Params::" + symbolProvider.toSymbol(outputShape).getName());
```

Comment:
I think "params_class" is fine here. There's nothing special about it being a stub.